src/components/utils/src/signals_linux.cc

```diff
@@ -37,8 +37,29 @@
 
 #include "utils/signals.h"
 
+void SigHandler(int sig) {
+  switch (sig) {
+    case SIGINT:
+      ::utils::handleSigs(NULL, "SIGINT signal has been caught");
+      break;
+    case SIGTERM:
+      ::utils::handleSigs(NULL, "SIGTERM signal has been caught");
+      break;
+    case SIGSEGV:
+      ::utils::handleSigs(NULL, "SIGSEGV signal has been caught");
+      break;
+    default:
+      ::utils::handleSigs(NULL, "Unexpected signal has been caught");
+      break;
+  }
+}
+}  //  namespace
+
 namespace utils {
 
+void handleSigs(void* signal_handle, const char* log_name) {}
+void WaitForSdlExecute() {}
+
 bool SubscribeToInterruptSignal(sighandler_t func) {
```

Comment:
you've removed this functions from header so remove them from here or move to anonymous namespace